tools/ci-build/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -10,12 +10,12 @@
 //! version numbers in addition to the dependency path.
 
 use crate::fs::Fs;
-use crate::package::{discover_manifests, parse_version};
+use crate::package::{discover_manifests, parse_version, SemVer};
 use crate::SDK_REPO_NAME;
 use anyhow::{bail, Context, Result};
 use clap::Parser;
-use semver::Version;
 use smithy_rs_tool_common::ci::running_in_ci;
+use smithy_rs_tool_common::package::SDK_PREFIX;
```

Comment:
I'm not sure why this constant is public... use `PackageCategory` instead:
```rust
PackageCategory::from_package_name(dep_name) == PackageCategory::AwsSdk
```

Reply:
Updated in b1ff4b6.